kubernetes-tests/src/test/java/io/fabric8/openshift/client/server/mock/DeploymentConfigTest.java

```diff
@@ -180,4 +182,11 @@ public void testDeployingLatestHandlesMissingLatestVersion() {
     assertNotNull(deploymentConfig);
     assertEquals(new Long(1), deploymentConfig.getStatus().getLatestVersion());
   }
+
+  @Test
```

Comment:
I did not get the point of this. I think we have already added a test for this. Secondly, if this is a test that is required, then no assert is there.

Reply:
Oops, This got added by accident.sorry